libcontainer/cgroups/fs2/hugetlb.go

```diff
@@ -43,19 +43,19 @@ func statHugeTlb(dirPath string, stats *cgroups.Stats) error {
 		usage := strings.Join([]string{"hugetlb", pagesize, "current"}, ".")
 		value, err := fscommon.GetCgroupParamUint(dirPath, usage)
 		if err != nil {
-			return errors.Wrapf(err, "failed to parse hugetlb.%s.current file", pagesize)
+			return errors.Wrap(err, "failed to parse "+usage)
```

Comment:
Actually, I think this could be just

```suggestion
			return err
```

As `GetCgroupParamUint` already includes a lot of details; https://github.com/opencontainers/runc/blob/5cc0deaf7a089a91a5ce4b81f835b64fcc4778d6/libcontainer/cgroups/fscommon/utils.go#L58-L70

Reply:
Makes sense, updated.
```diff
--- a/libcontainer/cgroups/fs2/hugetlb.go
+++ b/libcontainer/cgroups/fs2/hugetlb.go
@@ -43,7 +43,7 @@ func statHugeTlb(dirPath string, stats *cgroups.Stats) error {
                usage := strings.Join([]string{"hugetlb", pagesize, "current"}, ".")
                value, err := fscommon.GetCgroupParamUint(dirPath, usage)
                if err != nil {
-                       return errors.Wrap(err, "failed to parse "+usage)
+                       return err
                }
                hugetlbStats.Usage = value
 
@@ -55,7 +55,7 @@ func statHugeTlb(dirPath string, stats *cgroups.Stats) error {
                }
                _, value, err = fscommon.GetCgroupParamKeyValue(string(contents))
                if err != nil {
-                       return errors.Wrapf(err, "failed to parse "+fileName)
+                       return errors.Wrap(err, "failed to parse "+fileName)
                }
                hugetlbStats.Failcnt = value
```